wordpress-mobile/WordPress-Android
# Fix/prologue strings

This PR cherry picks the work @theck13 did on #13284 and port it to the release branch so that the fix will be added to the next beta and the strings sent to translators after the merge.

It updates the string resources used for the third page of the prologue screen from HTML tags to HTML entities in order to avoid missing text in strings sent to translators. 

**To test** (From #13284):
- Clear app data.
- Notice prologue screen is shown.
- Swipe to third page of prologue screen.
- Notice third page appears as After screenshot above.

**Review**
Only one developer is required to review these changes, but anyone can perform the review.

PR submission checklist:
